dynawo/sources/Launcher/launcher.in

```diff
@@ -26,6 +30,23 @@ if [[ -z "${DYNAWO_RESOURCES_DIR}" ]]; then
   export DYNAWO_RESOURCES_DIR=${DYNAWO_INSTALL_DIR}/@SHAREDIR_NAME@/
 fi
 export DYNAWO_XSD_DIR=${DYNAWO_INSTALL_DIR}/@SHAREDIR_NAME@/xsd/
+
+export IIDM_XML_XSD_PATH=${DYNAWO_LIBIIDM_INSTALL_DIR}/@SHAREDIR_NAME@/iidm/xsd/
+export IIDM_EXT_HVDCOPERATORACTIVEPOWERRANGE_XML_XSD_PATH=${DYNAWO_LIBIIDM_INSTALL_DIR}/@SHAREDIR_NAME@/iidm/xsd/
```

Comment:
If we now assume that dynawo iidm library diverges from the official one, why creating 15 environment variable that are all equals and all need to be set whenever dynawo is launched instead of creating a single one?

Reply:
Done